lib/modules/datasource/pypi/index.ts

```diff
@@ -82,6 +86,21 @@ export class PypiDatasource extends Datasource {
     return dependency;
   }
 
+  private async getAuthHeaders(
+    lookupUrl: string,
+  ): Promise<OutgoingHttpHeaders> {
+    const url = new URL(lookupUrl);
```

Comment:
Do we know 100% for sure that `lookupUrl` is validated as a URL by here? And if not then does `new URL()` throw, or return null?

Reply:
Good catch, I wrapped it in a try-catch